samples/client/petstore/java/okhttp-gson/README.md

```diff
@@ -277,11 +277,6 @@ Authentication schemes defined for the API:
 
 - **Type**: HTTP basic authentication
 
-<a id="http_signature_test"></a>
-### http_signature_test
-
-- **Type**: HTTP basic authentication
```

Comment:
note for discussion: when http_signature_test is kept, the samples contain this incorrect line of documentation